tests/system/test_dbapi.py

```diff
@@ -454,6 +456,49 @@ def test_read_timestamp_client_side_autocommit(self):
         read_timestamp_query_result_2 = self._cursor.fetchall()
         assert read_timestamp_query_result_1 != read_timestamp_query_result_2
 
+    @pytest.mark.parametrize("auto_commit", [False, True])
+    def test_batch_ddl(self, auto_commit, dbapi_database):
+        """Test batch ddl."""
+
+        self._conn._buffer_ddl_statements = False
+        if auto_commit:
```

Comment:
Can this be simplified to `self._conn.autocommit = auto_commit`

Reply:
Done